src/multi_channel_mosse.py

```diff
@@ -0,0 +1,146 @@
+import numpy as np
+from scipy.fftpack import fft2, ifft2, fftshift, ifftshift
+from scipy.stats import multivariate_normal
+from cvl.dataset import BoundingBox
+import cv2
+from cvl.image_io import crop_patch
+from mosse import MOSSETrackerGrayscale
+
+
+def get_multichannel_tracker(args):
+    return MOSSEMultiChannel(learning_rate=args.learning_rate, std=args.std)
+
+
+def default_descriptor(image):
+    return image
+
+def gradient_descriptor(image):
+    image = np.sum(image, axis=2) / 3
+    laplacian = cv2.Laplacian(image,cv2.CV_64F)[:,:,np.newaxis]
+    sobelx = cv2.Sobel(image,cv2.CV_64F,1,0,ksize=5)[:,:,np.newaxis]
+    sobely = cv2.Sobel(image,cv2.CV_64F,0,1,ksize=5)[:,:,np.newaxis]
+    return np.abs(np.concatenate([sobelx, sobely, laplacian], axis=2))
```

Comment:
Maybe try adding the grayscale image here as well?

Reply:
Could be worth trying! I'll test that